mne_qt_browser/_pg_figure.py

```diff
@@ -1339,8 +1339,9 @@ class VLineLabel(InfLineLabel):
     """Label of the vline displaying the time."""
 
     def __init__(self, vline):
+        vline = weakref.ref(vline)
         super().__init__(vline, text='{value:.3f} s', position=0.98,
-                         fill='g', color='b', movable=True)
+                         fill=(0, 191, 0), color='k', movable=True)
```

Comment:
The color definition should probably go into a constant so it can be easily re-used